desktop/src/main/java/bisq/desktop/main/dao/governance/make/MakeProposalView.java

```diff
@@ -300,19 +305,29 @@ private void publishMyProposal(ProposalType type) {
     }
 
     private void doPublishMyProposal(Proposal proposal, Transaction transaction) {
+
+        busyLabel.setVisible(true);
+        busyAnimation.play();
+        makeProposalButton.setDisable(true);
+
         daoFacade.publishMyProposal(proposal,
                 transaction,
                 () -> {
                     if (!DevEnv.isDevMode())
                         new Popup<>().feedback(Res.get("dao.tx.published.success")).show();
+
+                    if (proposalDisplay != null)
+                        proposalDisplay.clearForm();
+                    proposalTypeComboBox.getSelectionModel().clearSelection();
+                    busyAnimation.stop();
+                    busyLabel.setVisible(false);
+                    makeProposalButton.setDisable(false);
                 },
-                errorMessage -> new Popup<>().warning(errorMessage).show());
+                errorMessage -> {
+                    new Popup<>().warning(errorMessage).show();
+                    makeProposalButton.setDisable(false);
```

Comment:
By not disabling the busyAnimation/button here it will keep spinning after an error was reported, I don't think that's correct.

Reply:
Yes, it also should be reset in case of an error. Just pushed the commit.